src/stories/components/AlertModal/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+
+import * as S from './styles';
+import { AlertModalProps } from './types';
+
+export function AlertModal({ alertTitle, alertText }: AlertModalProps) {
+  const [open, setOpen] = useState(true);
+
+  const handleCloseModal = () => {
+    setOpen(false);
+  };
+
+  return (
+    <S.BoxModal title={alertTitle} closeModal={handleCloseModal} open={open}>
+      <S.BoxAlert>
+        {alertText}
+        <S.IconButton onClick={handleCloseModal} variant="outlined">
+          OK
+        </S.IconButton>
```

Comment:
Sugiro colcoar uma variavel pro titulo do botão (opcional); caso o usuario não passe nada, utilizar o texto OK.

```suggestion
        <S.IconButton onClick={handleCloseModal} variant="outlined">
          {buttonText ?? 'OK'}
        </S.IconButton>
```